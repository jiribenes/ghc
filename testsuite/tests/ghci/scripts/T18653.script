:set -XDataKinds -XUnboxedTuples
:kind! '(#,,,#) Int Char Bool Maybe
:kind! '(,,,) Int Char Bool Maybe
