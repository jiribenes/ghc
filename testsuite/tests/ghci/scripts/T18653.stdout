'(#,,,#) Int Char Bool Maybe :: (# *, *, *, * -> * #)
= '(# Int, Char, Bool, Maybe #)
'(,,,) Int Char Bool Maybe :: (*, *, *, * -> *)
= '(Int, Char, Bool, Maybe)
